Makefile.am

```diff
@@ -1,5 +1,5 @@
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = src man lib/gitsh vendor spec
+SUBDIRS = src man vendor spec ext/gitsh lib/gitsh
```

Comment:
I don't believe this matters. Not sure why I changed it now (I did the initial setup for this branch more than a year ago)